packages/boxel/addon/components/boxel/action-chin/index.gts

```diff
@@ -151,7 +153,7 @@ export default class ActionChin extends Component<Signature> {
       {{/if}}
       {{!-- in any state except: disabled + has a step, we want the lock shown --}}
       {{!-- template-lint-disable simple-unless --}}
-      {{#unless (and @disabled @stepNumber)}}
+      {{#unless (or (and @disabled @stepNumber) @hideLock)}}
```

Comment:
```suggestion
      {{#unless (or @hideLock (and @disabled @stepNumber))}}
```

This seems slightly easier to understand to me. Does the explanation two lines above need to be updated?